utils/bcm27xx-eeprom/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2020 OpenWrt.org
```

Comment:
Either assign copyright to yourself (or your company/employer), or omit the copyright notice (see [CONTRIBUTING.md](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain)).

Reply:
Ok